src/index.js

```diff
@@ -1,9 +1,11 @@
-/* eslint-disable no-unused-vars */
+/* eslint-disable */ /*importing files from the modules*/
 import _ from 'lodash';
 import './index.css';
-import populateTodoList from './print.js';
+import { saveTasks, addTasks, showTasks, tasksForm } from './print.js';
 
 document.addEventListener('DOMContentLoaded', () => {
-  populateTodoList();
+  saveTasks();
+  showTasks();
+  tasksForm.addEventListener('submit', addTasks);
+  tasksForm.addEventListener('submit', showTasks);
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md#:~:text=Project%20requirements-,Add%20a%20new%20JavaScript%20file%20and%20import%20it%20as%20a%20module,-%3A), Please add methods related to the status updates (completed: true / false) in a new module.

- [ ] Kindly add an event listener to the checkbox (change) that updates the items object's value for completed key upon user actions.

- [ ] Please Implement a function for the "Clear all completed" button (use filter() method).